apis/types/container_config.go

```diff
@@ -35,6 +35,9 @@ type ContainerConfig struct {
 	// Command to run specified an array of strings.
 	Cmd []string `json:"Cmd"`
 
+	// The ID of the container
+	ContainerID string `json:"ContainerID,omitempty"`
```

Comment:
`ContainerID` is used to be as `SandboxID`? I think it's a little confusing in the API level. Could we use annotation to handle this part?  
cc @YaoZengzeng @Starnop